apps/tunes/urls.py

```diff
@@ -8,4 +8,5 @@
     path('vote/', views.VoteView.as_view(), name='vote'),
     path('playlist/', views.PlaylistView.as_view(), name='playlist'),
     path('options/', views.OptionView.as_view(), name='options'),
+    path('last/', views.LastPlaylistView.as_view(), name='last')
```

Comment:
Let's make this `/browse/last/` its more appropriate for the pattern we're using for the tunes API endpoints